README.adoc

```diff
@@ -1,122 +1,337 @@
+[upperalpha]
+.. _If your docs project is its own Git repo_, change the boilerplate project's remote origin Git repository to your own.
 +
+.Example
+[source,shell]
 ----
 git remote set-url origin git://github.com:your-project/your-repo-name.git
 ----
 +
 Replace this URI with your own.
 
-.. _If your docs project is part of a parent directory's Git repo_, remove local Git files so your new docs source will be absorbed into the parent repo.
+.. _If your docs project is part of a parent directory's Git repo_, remove _local_ Git files (for the LDCMF repo you just cloned) so your new docs source will be absorbed into the parent repo.
 +
+.Example
+[source,shell]
 ----
 rm -rf .git
 ----
 +
 Now your new directory/files will appear as uncommitted additions to the parent project (`git status`).
 
-. Run Bundler to install dependencies.
+. *Run Bundler* to install dependencies.
 +
+[source,shell]
 ----
 bundle
 ----
 +
 If an error indicates Bundler is not installed, run `gem install bundler`, then repeat this command.
 
-. Rename this README so _init_ does not overwrite it.
+. *Rename this README* so the upcoming _init_ operation does not overwrite it.
 +
 .Example
+[source,shell]
 ----
 mv README.adoc README.meta.adoc
 ----
 +
 This file may confuse future users; we advise removing it before sharing your LDCMF project.
 
-. Edit your project metafile.
+. *Edit the metafile* for your new LDCMF project.
 +
 In your favorite text/code editor, open `data/meta.yml` and fill out the required info, then *save the file*.
 
-. Run LiquiDoc with the LDCMF initialization config.
+. *Initialize your custom environment.*
+[lowerroman]
+.. Generate an _initialization config_.
 +
+[source,shell]
 ----
-bundle exec liquidoc -c _init/init.yml
+bundle exec liquidoc --template _init/templates/init.yaml --data data/meta.yml --output _init/init.yml
 ----
 +
-This procedure customizes a few starter files based on `meta.yml`.
+This command invokes barest LiquiDoc, which performs a one-off template parsing.
 
-. You can now review your `README.adoc` and `content/index.adoc` files, editing and populating them with custom content.
+.. Generate _infrastructure files_ based on your custom preferences.
++
+[source,shell]
+----
+bundle exec liquidoc -c _init/init.yml
+----
+
+=== Build the Docs
+
+. You can now *review the generated `README.adoc`*. +
+The new README file is oriented for _your LDCMF application's_ future users (and you!).
+
+. *Perform the first build of your seedling docset*, as instructed in the new README. +
 
-. Delete or move the `README.meta.adoc` file (this file) and the `_init/` directory, which you will no longer need.
 
-Further instructions are in your new README!
-The new README file is oriented for your project's future users (and you!), so you don't need to repeat setup steps.
-Just run your first build.
+=== Wrapup
+
+. When you're certain everything looks good, *delete the `_init/` directory.
```

Comment:
- As a new user, how specifically can I feel comfortable that everything looks good?
- And how do I go check it out?
- Also, do we want to include a build and serve command instead, and then instruct users to check that address?
- Let's also specifically call out the fact that it automatically generates a PDF and an HTML version of the docs for this doc project.
- Missing end `*`

Reply:
There is no serve yet. The Jekyll part is rudimentary, because there are just too many ways that could be implemented and I haven't generalized it yet. It preps a lot of the Jekyll but it doesn't have render actions configured, so there's nothing to serve. That will probably come in v4.

The rest is all great points. I'll make it more explicit.